packages/flutter/test/material/data_table_test.dart

```diff
@@ -12,6 +12,7 @@ import 'package:vector_math/vector_math_64.dart' show Matrix3;
 
 import '../rendering/mock_canvas.dart';
```

Comment:
Before and after:
<img width="400" alt="dt_before" src="https://user-images.githubusercontent.com/7915388/103946328-4e87ee80-50eb-11eb-8084-c2e18491749b.png">
<img width="500" alt="dt_after" src="https://user-images.githubusercontent.com/7915388/103946330-4fb91b80-50eb-11eb-9568-fb0962c67601.png">